tests/buyProduct_test.js

```diff
@@ -1,5 +1,10 @@
 const orderHistory = require("../pages/orderHistory");
```

Comment:
remove  this line

Reply:
> remove this line

this line need, without it code does not work.
![Screenshot_1](https://user-images.githubusercontent.com/113456684/209129979-29cbfb6c-33f2-463a-92a9-bbd74bddc7a0.png)

Reply:
use `orderHistoryPage` in test